player.html

```diff
@@ -93,90 +94,89 @@
       text-align: center;
       cursor: pointer;
     }
-    </style>
-  </head>
-  <body>
-    <div id="game_container">
-      <div id="game">
-        <div data-bind="visible: disabled" id="nohost">NO HOST CONNECTED</div>
-        <ul id="ladder" data-bind="foreach: reversedSteps">
-          <li data-bind="text: $data, class: $parent.valueStatus($index)"></li>
-        </ul>
-        <div id="bank" data-bind="text: bankAmountThisRound"></div>
+  </style>
+</head>
+<body>
+  <div id="game_container">
+    <div id="game">
+      <div v-show="notConnected" id="nohost">NO HOST CONNECTED</div>
+      <ul id="ladder">
+        <li v-for="(step, index) in steps.slice().reverse()" v-bind:class="isCurrentAmount(step)">{{ step }}</li>
+      </ul>
+      <div id="bank" >{{ bankAmountThisRound }}</div>
 
-        <div id="timer" data-bind="text: timeLeft"></div>
-      </div>
-      <div id="audio_blocked_indicator" data-bind="if: audioBlocked">Cliquer ici pour activer l'audio</div>
+      <div id="timer">{{ timeLeft }}</div>
     </div>
-    <script src="/socket.io/socket.io.js"></script>
-    <script>
+    <div id="audio_blocked_indicator" v-if="audioBlocked">Cliquer ici pour activer l'audio</div>
+  </div>
+  <script src="/socket.io/socket.io.js"></script>
+  <script>
     const urlParams = new URLSearchParams(window.location.search);
     const gameId = urlParams.get('game_id');
 
-    let state = {
-      steps: ko.observable([]),
-      bankAmountThisRound: ko.observable(0),
-      currentAmount: ko.observable(0),
-      currentStep: ko.observable(0),
-      timeLeft: ko.observable(""),
-      currentPlayer: ko.observable(""),
-      disabled: ko.observable(false),
-      audioBlocked: ko.observable(false),
-    };
-    state.reversedSteps = ko.computed(function() {
-      return this.steps().slice(0).reverse();
-    }, state);
-    state.valueStatus = function(index) {
-      return (this.steps().length - this.currentStep() - 1) <= index() ? "active" : "";
-    };
-
-    // Activates knockout.js
-    ko.applyBindings(state, document.body);
+    // Instanciate Vue
+    const vm = new Vue({
+      el: '#game_container',
+      data: {
+        notConnected: true, 
+        steps: [],
+        bankAmountThisRound: 0,
+        timeLeft: 0,
+        audioBlocked: true
+      },
+      methods: {
+        isCurrentAmount(step) {
```

Comment:
Functions starting with 'is' sound like they'll return a boolean.

Maybe getClassForAmount instead?